Readme-LavaInstallation.md

```diff
@@ -224,12 +236,24 @@ cat <<EOT >> /etc/samba/smb.conf
    guest ok = yes
 EOT
 service smbd restart
+```
+To verify if the SMB share is working type this:
+```bash
 mount //$(hostname)/lava /var/lib/lava/dispatcher/tmp
 # press Enter (as there is an empty password)
 ```
 
+Go to http://your-lava-machine-ip/admin/lava_scheduler_app/device/localhost/ and don't forget to change your-lava-machine-ip into an actual ip.
```

Comment:
Go to http://<LAVA-SLAVE>/admin/lava_scheduler_app/device/localhost/ and change the "device type" to 'hyperv'

this line is not required.